subsys/testsuite/ztest/Kconfig

```diff
@@ -91,6 +91,39 @@ config ZTEST_RULE_1CPU
 
 endmenu
 
+config ZTEST_DO_THE_SHUFFLE
+	bool "Enables the shuffling of tests and suites"
+	default n
+	help
+	  This rule will shuffle the order of tests and test suites.
+
+if ZTEST_DO_THE_SHUFFLE
+config ZTEST_DO_THE_SHUFFLE_SEED
+	int "Random number generator seed value"
+	default 0
```

Comment:
I thought you randomized the seed?

Reply:
Missed adding that file.  Let me upload that change